packages/gatsby-source-drupal/src/normalize.js

```diff
@@ -55,6 +55,11 @@ const createNodeIdWithVersion = (
   revisionId,
   entityReferenceRevisions = []
 ) => {
+  // Fallback to default language for entities that don't translate.
+  if (getOptions().languageConfig.nonTranslatableEntities.includes(type)) {
```

Comment:
Something like this might work:

```
if (getOptions()?.languageConfig.nonTranslatableEntities.includes(type)) {
```

Reply:
@smthomas - made suggested change.

Reply:
@smthomas - the only thing left failing appears to be something unrelated to this change. Not sure how that would have passed previously...

Reply:
That's failing on all tests atm so I'll approve and get this in. Thanks!